rclpy/rclpy/__init__.py

```diff
@@ -97,37 +97,40 @@ def spin_once(node, *, timeout_sec=None):
         executor.remove_node(node)
 
 
-def spin(node):
+def spin(node, executor=None):
```

Comment:
Should we also be optionally passing the executor to `spin_once()`?

Reply:
Yeah, good idea, I added that in 04398ba